2025-08-23 21:50:09,909 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) drop table if exists parcela cascade 
2025-08-23 21:50:09,909 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: produtos) drop table dbo.PRODUTO
2025-08-23 21:50:09,917 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) drop table if exists resultado_simulacao cascade 
2025-08-23 21:50:09,917 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) drop table if exists simulacao cascade 
2025-08-23 21:50:09,919 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [org.hib.too.sch.int.ExceptionHandlerLoggedImpl] (JPA Startup Thread: produtos) GenerationTarget encountered exception accepting command : Error executing DDL "drop table dbo.PRODUTO" via JDBC [Cannot drop the table 'dbo.PRODUTO', because it does not exist or you do not have permission.]: org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "drop table dbo.PRODUTO" via JDBC [Cannot drop the table 'dbo.PRODUTO', because it does not exist or you do not have permission.]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:93)
	at org.hibernate.tool.schema.internal.Helper.applySqlString(Helper.java:221)
	at org.hibernate.tool.schema.internal.Helper.applySqlStrings(Helper.java:207)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.dropTables(SchemaDropperImpl.java:383)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.dropConstraintsTablesSequences(SchemaDropperImpl.java:255)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.dropFromMetadata(SchemaDropperImpl.java:217)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.performDrop(SchemaDropperImpl.java:185)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:155)
	at org.hibernate.tool.schema.internal.SchemaDropperImpl.doDrop(SchemaDropperImpl.java:115)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:225)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:142)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:139)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:33)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:350)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:97)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:74)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:90)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:66)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:165)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot drop the table 'dbo.PRODUTO', because it does not exist or you do not have permission.
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:276)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1787)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:975)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:869)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7745)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:4700)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:321)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:253)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:842)
	at io.agroal.pool.wrapper.StatementWrapper.execute(StatementWrapper.java:220)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	... 22 more

2025-08-23 21:50:09,928 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) create table parcela (valorAmortizacao numeric(18,2) not null, valorJuros numeric(18,2) not null, valorPrestacao numeric(18,2) not null, id bigint generated by default as identity, numero bigint not null, resultado_simulacao_id bigint, primary key (id))
2025-08-23 21:50:09,928 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: produtos) create table dbo.PRODUTO (CO_PRODUTO int not null, NU_MAXIMO_MESES smallint, NU_MINIMO_MESES smallint not null, PC_TAXA_JUROS numeric(10,9) not null, VR_MAXIMO numeric(18,2), VR_MINIMO numeric(18,2) not null, NO_PRODUTO varchar(200) not null, primary key (CO_PRODUTO))
2025-08-23 21:50:09,942 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) create table resultado_simulacao (id bigint generated by default as identity, simulacao_id bigint, taxaMediaJuros bigint, valorMedioPrestacao bigint, valorTotalCredito bigint, valorTotalDesejado bigint, tipo enum ('PRICE','SAC') not null, primary key (id))
2025-08-23 21:50:09,945 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) create table simulacao (taxaMediaJuros numeric(10,4), valorDesejado numeric(18,2) not null, valorMedioPrestacao numeric(18,2), valorTotalCredito numeric(18,2), valorTotalDesejado numeric(18,2), dataSimulacao timestamp(6) not null, id bigint generated by default as identity, prazo bigint not null, primary key (id))
2025-08-23 21:50:09,948 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) alter table if exists parcela add constraint FK9wi92sjbxcv741a0eip840c9g foreign key (resultado_simulacao_id) references resultado_simulacao
2025-08-23 21:50:09,977 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) alter table if exists resultado_simulacao add constraint FK5qmkgq2er31aj9uo2i2jvhi2y foreign key (simulacao_id) references simulacao
2025-08-23 21:50:09,995 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: produtos) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (1, 'Empréstimo Pessoal Básico', 0.017900000, 6, 24, 1000.00, 50000.00)
2025-08-23 21:50:09,996 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (1, 'Produto 1', 0.017900000, 0, 24, 200.00, 10000.00)
2025-08-23 21:50:10,001 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: produtos) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (2, 'Empréstimo Pessoal Premium', 0.015500000, 12, 48, 10000.00, 200000.00)
2025-08-23 21:50:10,001 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [org.hib.too.sch.int.ExceptionHandlerLoggedImpl] (JPA Startup Thread: <default>) GenerationTarget encountered exception accepting command : Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (1, 'Produto 1', 0.017900000, 0, 24, 200.00, 10000.00)" via JDBC [Table "PRODUTO" not found;]: org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (1, 'Produto 1', 0.017900000, 0, 24, 200.00, 10000.00)" via JDBC [Table "PRODUTO" not found;]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:93)
	at org.hibernate.tool.schema.internal.Helper.applySqlString(Helper.java:221)
	at org.hibernate.tool.schema.internal.Helper.applyScript(Helper.java:244)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportFiles(AbstractSchemaPopulator.java:150)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportSources(AbstractSchemaPopulator.java:58)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.performCreation(SchemaCreatorImpl.java:172)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:130)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:106)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:237)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:142)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:139)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:33)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:350)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:97)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:74)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:90)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:66)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:165)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUTO" not found; SQL statement:
INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (1, 'Produto 1', 0.017900000, 0, 24, 200.00, 10000.00) [42102-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7944)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7915)
	at org.h2.command.Parser.readTableOrView(Parser.java:7894)
	at org.h2.command.Parser.readTableOrView(Parser.java:7856)
	at org.h2.command.Parser.parseInsert(Parser.java:1520)
	at org.h2.command.Parser.parsePrepared(Parser.java:697)
	at org.h2.command.Parser.parse(Parser.java:580)
	at org.h2.command.Parser.parse(Parser.java:555)
	at org.h2.command.Parser.prepareCommand(Parser.java:484)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:644)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:560)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1164)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at io.agroal.pool.wrapper.StatementWrapper.execute(StatementWrapper.java:220)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	... 21 more

2025-08-23 21:50:10,004 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (2, 'Produto 2', 0.017500000, 25, 48, 10001.00, 100000.00)
2025-08-23 21:50:10,005 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [org.hib.too.sch.int.ExceptionHandlerLoggedImpl] (JPA Startup Thread: <default>) GenerationTarget encountered exception accepting command : Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (2, 'Produto 2', 0.017500000, 25, 48, 10001.00, 100000.00)" via JDBC [Table "PRODUTO" not found;]: org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (2, 'Produto 2', 0.017500000, 25, 48, 10001.00, 100000.00)" via JDBC [Table "PRODUTO" not found;]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:93)
	at org.hibernate.tool.schema.internal.Helper.applySqlString(Helper.java:221)
	at org.hibernate.tool.schema.internal.Helper.applyScript(Helper.java:244)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportFiles(AbstractSchemaPopulator.java:150)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportSources(AbstractSchemaPopulator.java:58)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.performCreation(SchemaCreatorImpl.java:172)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:130)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:106)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:237)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:142)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:139)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:33)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:350)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:97)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:74)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:90)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:66)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:165)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUTO" not found; SQL statement:
INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (2, 'Produto 2', 0.017500000, 25, 48, 10001.00, 100000.00) [42102-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7944)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7915)
	at org.h2.command.Parser.readTableOrView(Parser.java:7894)
	at org.h2.command.Parser.readTableOrView(Parser.java:7856)
	at org.h2.command.Parser.parseInsert(Parser.java:1520)
	at org.h2.command.Parser.parsePrepared(Parser.java:697)
	at org.h2.command.Parser.parse(Parser.java:580)
	at org.h2.command.Parser.parse(Parser.java:555)
	at org.h2.command.Parser.prepareCommand(Parser.java:484)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:644)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:560)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1164)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at io.agroal.pool.wrapper.StatementWrapper.execute(StatementWrapper.java:220)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	... 21 more

2025-08-23 21:50:10,008 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (3, 'Produto 3', 0.018200000, 49, 96, 100000.01, 1000000.00)
2025-08-23 21:50:10,009 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [org.hib.too.sch.int.ExceptionHandlerLoggedImpl] (JPA Startup Thread: <default>) GenerationTarget encountered exception accepting command : Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (3, 'Produto 3', 0.018200000, 49, 96, 100000.01, 1000000.00)" via JDBC [Table "PRODUTO" not found;]: org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (3, 'Produto 3', 0.018200000, 49, 96, 100000.01, 1000000.00)" via JDBC [Table "PRODUTO" not found;]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:93)
	at org.hibernate.tool.schema.internal.Helper.applySqlString(Helper.java:221)
	at org.hibernate.tool.schema.internal.Helper.applyScript(Helper.java:244)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportFiles(AbstractSchemaPopulator.java:150)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportSources(AbstractSchemaPopulator.java:58)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.performCreation(SchemaCreatorImpl.java:172)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:130)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:106)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:237)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:142)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:139)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:33)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:350)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:97)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:74)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:90)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:66)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:165)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUTO" not found; SQL statement:
INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (3, 'Produto 3', 0.018200000, 49, 96, 100000.01, 1000000.00) [42102-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7944)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7915)
	at org.h2.command.Parser.readTableOrView(Parser.java:7894)
	at org.h2.command.Parser.readTableOrView(Parser.java:7856)
	at org.h2.command.Parser.parseInsert(Parser.java:1520)
	at org.h2.command.Parser.parsePrepared(Parser.java:697)
	at org.h2.command.Parser.parse(Parser.java:580)
	at org.h2.command.Parser.parse(Parser.java:555)
	at org.h2.command.Parser.prepareCommand(Parser.java:484)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:644)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:560)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1164)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at io.agroal.pool.wrapper.StatementWrapper.execute(StatementWrapper.java:220)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	... 21 more

2025-08-23 21:50:10,011 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: <default>) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (4, 'Produto 4', 0.015100000, 96, null, 1000000.01, null)
2025-08-23 21:50:10,010 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (JPA Startup Thread: produtos) INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (3, 'Crédito Consignado', 0.012000000, 24, 84, 5000.00, 300000.00)
2025-08-23 21:50:10,012 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [org.hib.too.sch.int.ExceptionHandlerLoggedImpl] (JPA Startup Thread: <default>) GenerationTarget encountered exception accepting command : Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (4, 'Produto 4', 0.015100000, 96, null, 1000000.01, null)" via JDBC [Table "PRODUTO" not found;]: org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (4, 'Produto 4', 0.015100000, 96, null, 1000000.01, null)" via JDBC [Table "PRODUTO" not found;]
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:93)
	at org.hibernate.tool.schema.internal.Helper.applySqlString(Helper.java:221)
	at org.hibernate.tool.schema.internal.Helper.applyScript(Helper.java:244)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportFiles(AbstractSchemaPopulator.java:150)
	at org.hibernate.tool.schema.internal.AbstractSchemaPopulator.applyImportSources(AbstractSchemaPopulator.java:58)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.performCreation(SchemaCreatorImpl.java:172)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:130)
	at org.hibernate.tool.schema.internal.SchemaCreatorImpl.doCreation(SchemaCreatorImpl.java:106)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:237)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:142)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:139)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:33)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:350)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:97)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:74)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:90)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:66)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:165)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PRODUTO" not found; SQL statement:
INSERT INTO PRODUTO (CO_PRODUTO, NO_PRODUTO, PC_TAXA_JUROS, NU_MINIMO_MESES, NU_MAXIMO_MESES, VR_MINIMO, VR_MAXIMO) VALUES (4, 'Produto 4', 0.015100000, 96, null, 1000000.01, null) [42102-230]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7944)
	at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:7915)
	at org.h2.command.Parser.readTableOrView(Parser.java:7894)
	at org.h2.command.Parser.readTableOrView(Parser.java:7856)
	at org.h2.command.Parser.parseInsert(Parser.java:1520)
	at org.h2.command.Parser.parsePrepared(Parser.java:697)
	at org.h2.command.Parser.parse(Parser.java:580)
	at org.h2.command.Parser.parse(Parser.java:555)
	at org.h2.command.Parser.prepareCommand(Parser.java:484)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:644)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:560)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1164)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:245)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:231)
	at io.agroal.pool.wrapper.StatementWrapper.execute(StatementWrapper.java:220)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:79)
	... 21 more

2025-08-23 21:50:10,186 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.quarkus] (Quarkus Main Thread) simulacao-emprestimo 1.0.0-SNAPSHOT on JVM (powered by Quarkus 3.25.3) started in 33.002s. Listening on: http://localhost:8080
2025-08-23 21:50:10,186 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.quarkus] (Quarkus Main Thread) Profile dev activated. Live Coding activated.
2025-08-23 21:50:10,187 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.quarkus] (Quarkus Main Thread) Installed features: [agroal, cache, cdi, compose, flyway, hibernate-orm, hibernate-orm-panache, hibernate-validator, jdbc-h2, jdbc-mssql, narayana-jta, rest, rest-jackson, smallrye-context-propagation, smallrye-health, smallrye-openapi, swagger-ui, vertx]
2025-08-23 21:50:59,726 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] ERROR [res.exc.GenericExceptionMapper] (vert.x-eventloop-thread-0) Erro não tratado capturado: NotFoundException - Path: /: jakarta.ws.rs.NotFoundException: HTTP 404 Not Found
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.handle(RestInitialHandler.java:70)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:121)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.beginProcessing(RestInitialHandler.java:48)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:23)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:10)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:62)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:40)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:379)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:368)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:164)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:94)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$5.handle(VertxHttpHotReplacementSetup.java:206)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup$5.handle(VertxHttpHotReplacementSetup.java:194)
	at io.vertx.core.impl.future.FutureImpl$4.onSuccess(FutureImpl.java:176)
	at io.vertx.core.impl.future.FutureBase.lambda$emitSuccess$0(FutureBase.java:60)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:998)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)

2025-08-23 21:50:59,846 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] ERROR [res.exc.GenericExceptionMapper] (vert.x-eventloop-thread-0) Erro não tratado capturado: NotFoundException - Path: /: jakarta.ws.rs.NotFoundException: HTTP 404 Not Found
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.handle(RestInitialHandler.java:70)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:121)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.beginProcessing(RestInitialHandler.java:48)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:23)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:10)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:62)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:40)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:379)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:368)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:164)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:94)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup.handleHotReplacementRequest(VertxHttpHotReplacementSetup.java:146)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$HotReplacementRoutingContextHandler.handle(VertxHttpRecorder.java:1717)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$HotReplacementRoutingContextHandler.handle(VertxHttpRecorder.java:1707)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.vertx.ext.web.impl.RoutingContextImpl.route(RoutingContextImpl.java:100)
	at io.vertx.ext.web.impl.RouterImpl.handle(RouterImpl.java:69)
	at io.vertx.ext.web.impl.RouterImpl.handle(RouterImpl.java:37)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$7.handle(VertxHttpRecorder.java:540)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$7.handle(VertxHttpRecorder.java:537)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$2.handle(HttpServerCommonHandlers.java:87)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$2.handle(HttpServerCommonHandlers.java:70)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$1.handle(VertxHttpRecorder.java:186)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$1.handle(VertxHttpRecorder.java:160)
	at io.vertx.core.impl.ContextImpl.emit(ContextImpl.java:342)
	at io.vertx.core.impl.DuplicatedContext.emit(DuplicatedContext.java:163)
	at io.vertx.core.http.impl.Http1xServerConnection.handleMessage(Http1xServerConnection.java:174)
	at io.vertx.core.net.impl.ConnectionBase.read(ConnectionBase.java:159)
	at io.vertx.core.net.impl.VertxHandler.channelRead(VertxHandler.java:153)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:289)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:796)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:732)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:998)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)

2025-08-23 21:51:00,383 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] ERROR [res.exc.GenericExceptionMapper] (vert.x-eventloop-thread-0) Erro não tratado capturado: NotFoundException - Path: /favicon.ico: jakarta.ws.rs.NotFoundException: HTTP 404 Not Found
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.handle(RestInitialHandler.java:70)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:121)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at org.jboss.resteasy.reactive.server.handlers.RestInitialHandler.beginProcessing(RestInitialHandler.java:48)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:23)
	at org.jboss.resteasy.reactive.server.vertx.ResteasyReactiveVertxHandler.handle(ResteasyReactiveVertxHandler.java:10)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:62)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$1.handle(HttpServerCommonHandlers.java:40)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:379)
	at io.quarkus.resteasy.reactive.server.runtime.ResteasyReactiveRecorder$13.handle(ResteasyReactiveRecorder.java:368)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:164)
	at io.quarkus.vertx.http.runtime.filters.accesslog.AccessLogHandler.handle(AccessLogHandler.java:94)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.quarkus.vertx.http.runtime.devmode.VertxHttpHotReplacementSetup.handleHotReplacementRequest(VertxHttpHotReplacementSetup.java:146)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$HotReplacementRoutingContextHandler.handle(VertxHttpRecorder.java:1717)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$HotReplacementRoutingContextHandler.handle(VertxHttpRecorder.java:1707)
	at io.vertx.ext.web.impl.RouteState.handleContext(RouteState.java:1283)
	at io.vertx.ext.web.impl.RoutingContextImplBase.iterateNext(RoutingContextImplBase.java:177)
	at io.vertx.ext.web.impl.RoutingContextImpl.next(RoutingContextImpl.java:151)
	at io.vertx.ext.web.impl.RoutingContextImpl.route(RoutingContextImpl.java:100)
	at io.vertx.ext.web.impl.RouterImpl.handle(RouterImpl.java:69)
	at io.vertx.ext.web.impl.RouterImpl.handle(RouterImpl.java:37)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$7.handle(VertxHttpRecorder.java:540)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$7.handle(VertxHttpRecorder.java:537)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$2.handle(HttpServerCommonHandlers.java:87)
	at io.quarkus.vertx.http.runtime.options.HttpServerCommonHandlers$2.handle(HttpServerCommonHandlers.java:70)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$1.handle(VertxHttpRecorder.java:186)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder$1.handle(VertxHttpRecorder.java:160)
	at io.vertx.core.impl.ContextImpl.emit(ContextImpl.java:342)
	at io.vertx.core.impl.DuplicatedContext.emit(DuplicatedContext.java:163)
	at io.vertx.core.http.impl.Http1xServerConnection.handleMessage(Http1xServerConnection.java:174)
	at io.vertx.core.net.impl.ConnectionBase.read(ConnectionBase.java:159)
	at io.vertx.core.net.impl.VertxHandler.channelRead(VertxHandler.java:153)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:289)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1357)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:868)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:796)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:732)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:998)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)

2025-08-23 21:51:13,381 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [res.SimulacaoResource] (executor-thread-3) [requestId=3e63809f-bb34-48af-97c6-43052b95184b] Iniciando criação de simulação - Valor: 900000000, Prazo: 5 meses
2025-08-23 21:51:13,669 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] DEBUG [org.hib.SQL] (executor-thread-3) select p1_0.CO_PRODUTO,p1_0.NO_PRODUTO,p1_0.NU_MAXIMO_MESES,p1_0.NU_MINIMO_MESES,p1_0.PC_TAXA_JUROS,p1_0.VR_MAXIMO,p1_0.VR_MINIMO from dbo.PRODUTO p1_0
2025-08-23 21:51:13,737 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [res.exc.ProdutoExceptionMapper] (executor-thread-3) Erro relacionado a produto: PRODUCTS_NOT_ELIGIBLE - Path: /v1/simulacoes - Detalhe: Valor: R$ 900000000.00, Prazo: 5 meses
2025-08-23 21:51:23,110 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [res.SimulacaoResource] (executor-thread-3) [requestId=f2cbdb7c-a403-40da-8b94-12b9898caf8c] Iniciando criação de simulação - Valor: 900, Prazo: 5 meses
2025-08-23 21:51:23,112 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [res.exc.ProdutoExceptionMapper] (executor-thread-3) Erro relacionado a produto: PRODUCTS_NOT_ELIGIBLE - Path: /v1/simulacoes - Detalhe: Valor: R$ 900.00, Prazo: 5 meses
2025-08-23 21:51:34,261 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] ERROR [res.exc.GenericExceptionMapper] (executor-thread-3) Erro não tratado capturado: WebApplicationException - Path: /v1/simulacoes: jakarta.ws.rs.WebApplicationException: HTTP 400 Bad Request
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:66)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:83)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader_ClientProxy.readFrom(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.RequestDeserializeHandler.readFrom(RequestDeserializeHandler.java:126)
	at org.jboss.resteasy.reactive.server.handlers.RequestDeserializeHandler.handle(RequestDeserializeHandler.java:84)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:135)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$15.runWith(VertxCoreRecorder.java:645)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('}' (code 125)): was expecting double-quote to start field name
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 4, column: 1]
	at com.fasterxml.jackson.core.JsonParser._constructReadException(JsonParser.java:2672)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:742)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._handleOddName(UTF8StreamJsonParser.java:2103)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._parseName(UTF8StreamJsonParser.java:1753)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextFieldName(UTF8StreamJsonParser.java:1040)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:309)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:169)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2130)
	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1500)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.doReadFrom(ServerJacksonMessageBodyReader.java:93)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:44)
	... 15 more

2025-08-23 21:51:35,805 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] ERROR [res.exc.GenericExceptionMapper] (executor-thread-3) Erro não tratado capturado: WebApplicationException - Path: /v1/simulacoes: jakarta.ws.rs.WebApplicationException: HTTP 400 Bad Request
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:66)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:83)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader_ClientProxy.readFrom(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.RequestDeserializeHandler.readFrom(RequestDeserializeHandler.java:126)
	at org.jboss.resteasy.reactive.server.handlers.RequestDeserializeHandler.handle(RequestDeserializeHandler.java:84)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:135)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$15.runWith(VertxCoreRecorder.java:645)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('}' (code 125)): was expecting double-quote to start field name
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 4, column: 1]
	at com.fasterxml.jackson.core.JsonParser._constructReadException(JsonParser.java:2672)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:742)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._handleOddName(UTF8StreamJsonParser.java:2103)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._parseName(UTF8StreamJsonParser.java:1753)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextFieldName(UTF8StreamJsonParser.java:1040)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:309)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:169)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2130)
	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1500)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.doReadFrom(ServerJacksonMessageBodyReader.java:93)
	at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.ServerJacksonMessageBodyReader.readFrom(ServerJacksonMessageBodyReader.java:44)
	... 15 more

2025-08-23 21:51:50,017 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [res.SimulacaoResource] (executor-thread-3) [requestId=b0b1ab7b-542f-4499-b076-2d719d09541c] Iniciando criação de simulação - Valor: 900, Prazo: 6 meses
2025-08-23 21:51:50,019 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [res.exc.ProdutoExceptionMapper] (executor-thread-3) Erro relacionado a produto: PRODUCTS_NOT_ELIGIBLE - Path: /v1/simulacoes - Detalhe: Valor: R$ 900.00, Prazo: 6 meses
2025-08-23 21:51:55,789 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [res.SimulacaoResource] (executor-thread-3) [requestId=2403ba99-8f00-4c67-a47c-5d4da39b1a9b] Iniciando criação de simulação - Valor: 900, Prazo: 10 meses
2025-08-23 21:51:55,791 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [res.exc.ProdutoExceptionMapper] (executor-thread-3) Erro relacionado a produto: PRODUCTS_NOT_ELIGIBLE - Path: /v1/simulacoes - Detalhe: Valor: R$ 900.00, Prazo: 10 meses
2025-08-23 21:51:57,260 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [res.SimulacaoResource] (executor-thread-3) [requestId=d593843d-f22e-4ea8-ad1d-7409790375c6] Iniciando criação de simulação - Valor: 900, Prazo: 10 meses
2025-08-23 21:51:57,261 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] WARN  [res.exc.ProdutoExceptionMapper] (executor-thread-3) Erro relacionado a produto: PRODUCTS_NOT_ELIGIBLE - Path: /v1/simulacoes - Detalhe: Valor: R$ 900.00, Prazo: 10 meses
2025-08-23 21:53:03,153 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.qua.dep.dev.RuntimeUpdatesProcessor] (vert.x-worker-thread-1) File change detected: /home/silver/Documents/Hackaton/Hackaton-caixa-master/src/main/resources/import.sql
2025-08-23 21:53:03,156 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.qua.dep.dev.RuntimeUpdatesProcessor] (vert.x-worker-thread-1) Restarting quarkus due to changes in import.sql.
2025-08-23 21:53:03,191 silver-aspire-f5-573g /usr/lib/jvm/java-17-openjdk-amd64/bin/java[32364] INFO  [io.quarkus] (Quarkus Main Thread) simulacao-emprestimo stopped in 0.027s
